data/database/src/main/java/io/filmtime/data/database/MovieDetailEntity.kt

```diff
@@ -0,0 +1,18 @@
+package io.filmtime.data.database
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class MovieDetailEntity(
```

Comment:
I'd suggest to use table name for db entities, so Renaming the kotlin model does not affect the sqlite DB queries and table names
```suggestion
@Entity(tableName = "movie_detail")
data class MovieDetailEntity(
```